express/code/blocks/cta-carousel/cta-carousel.css

```diff
@@ -238,6 +242,7 @@
     position: absolute;
     height: 100%;
     width: 100%;
+    z-index: 9;
```

Comment:
is this needed? I don't see it in the page in the PR descriptions. Is it for a different page that we should check?

Reply:
Not needed, good catch. I was testing something else.

Reply:
Fixed @JingleH